node/src/tests.rs

```diff
@@ -1,10 +0,0 @@
-use crate::chain_spec::validate_endowment;
```

Comment:
do you not wanna move this to another file?

Reply:
I want to keep these tests in their relevant chain spec files. I expect we will have different types of checks for diff chain specs. For example the turing will also have tests on the vesting data